src/web/mjs/connectors/XCaliBRScans.mjs

```diff
@@ -0,0 +1,24 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class XCaliBRScans extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'xcalibrscans';
+        super.label = 'xCaliBR Scans';
```

Comment:
label capitalization difference with classname or something intended ?

Reply:
Intentional.
Class names are supposed to be PascalCase, but the label is spelled the same way as the scanlation group does on their website.